app-config.yaml

```diff
@@ -18,10 +18,13 @@ app:
 organization:
   name: Maven Wave
 
+permission:
+  enabled: true
```

Comment:
We need to do configuration in the auth backend to allow for this.

Reply:
that's right